src/Jabbr-Core.Data/Jabbr-Core.Data2.xproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@adamtuliper Did not touch .Data library. I'll let @ashanhol change these as needed after merge.